### ISSUE TITLE
Moving RleVec library to the official alternative container organization

### ISSUE THREAD

**Issue by Kerollmops** [veldsla/rle_vec#3]
Why not moving this library to the official [Contain-rs](https://github.com/contain-rs/) organization that is for extension of Rust's `std::collections` like they said ?

Of course we keep collaboration rights and access !

**Comment by veldsla** [comment]
It would be fine with me, repository location is not really that important is it? I also recently saw this: https://users.rust-lang.org/t/all-contain-rs-crates-are-now-in-maintenance-mode/12056

**Comment by Kerollmops** [comment]
Yes the actual ones, but this repo will not be in maintenance mode.
